src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.data.mongodb.uri=mongodb://localhost/test
```

Comment:
No need for the `/test` is there?  I think it's just thrown away.

Reply:
True. Removed.